lib/mcb/commands/apiv1/courses/find.rb

```diff
@@ -0,0 +1,62 @@
+summary 'Find a particular provider course entry'
+description <<~EODESCRIPTION
+  Searches for a course with the given provider and course code by iterating
+  through all the pages of results provided from the course endpoint, outputing
+  the found record.
```

Comment:
```suggestion
  the found records.
```

grammar?

Reply:
Maybe search-and-replace gone bad? Anyway, thanks!